tetris/piece.go

```diff
@@ -5,25 +5,26 @@ import (
 )
 
 const (
-    ColorReset  = "\033[0m"
-    ColorRed    = "\033[31m"
-    ColorGreen  = "\033[32m"
-    ColorYellow = "\033[33m"
-    ColorBlue   = "\033[34m"
-    ColorPurple = "\033[35m"
-    ColorCyan   = "\033[36m"
-    ColorWhite  = "\033[37m"
+	ColorReset  = "\033[0m"
+	ColorRed    = "\033[31m"
+	ColorGreen  = "\033[32m"
+	ColorYellow = "\033[33m"
+	ColorBlue   = "\033[34m"
+	ColorPurple = "\033[35m"
+	ColorCyan   = "\033[36m"
+	ColorWhite  = "\033[37m"
 )
 
+type Color string
 
-var pieceColors = []string{
-    ColorCyan,    
-    ColorYellow,  
-    ColorGreen,   
-    ColorRed,     
-    ColorBlue,    
-    ColorPurple,  
-    ColorWhite, 
+var pieceColors = []Color{
```

Comment:
Are we using this anywhere?

Reply:
I was using it to initialize the color field of each piece in the pieces map but now can see we don't need it as colors are already defined :)